backend/ticket/views.py

```
@@ -0,0 +1,41 @@
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+from django.contrib.auth import authenticate,login
```

Ruff (0.12.2)

`django.contrib.auth.authenticate` imported but unused

Remove unused import

(F401)